dotcom-rendering/src/lib/renderElement.tsx

```diff
@@ -92,6 +94,7 @@ type Props = {
 	totalElements?: number;
 	isListElement?: boolean;
 	isSectionedMiniProfilesArticle?: boolean;
+	tags?: TagType[];
```

Comment:
It looks like the tags are sent through several layers of components to be used in conjunction with `imageOverrideUrl` to decide which image should be used for the byline?

Would it make sense to instead update the `MultiByline` type to have an `imageUrl` field, instead of `imageOverrideUrl`, and derive that from the tags and the override URL much earlier, in the enhancer? That way you save on some data handling complexity in the component, allowing it just to focus on rendering, and you also reduce prop drilling through several other components?